server/util/ongoingRegistration.js

```diff
@@ -0,0 +1,32 @@
+const { parseSemesterIntoYearSemesterNumber, findMatchedPeriod, SEMESTER_NUMBER } = require('./semesterUtils')
+
+const computeFirstRegistrationDate = ladokSemester => {
+  const registrationDates = { [SEMESTER_NUMBER.SPRING]: '09-15', [SEMESTER_NUMBER.AUTUMN]: '03-15' }
+  const summerRegistrationDate = '02-15'
+  const registrationDateString = `${ladokSemester.year}-${ladokSemester.semesterNumber != 3 ? registrationDates[ladokSemester.semesterNumber] : summerRegistrationDate}`
```

Comment:
Suggested changes in combination with above changes:

```suggestion
  const registrationDateString = `${ladokSemester.year}-${REGISTRATION_DATES[ladokSemester.semesterNumber] 
```